components/ui/NavBar.tsx

```diff
@@ -0,0 +1,43 @@
+"use client"
+import React, { useState, useEffect } from 'react';
+
+
+const NavBar: React.FC = () => {
+  const [scrolled, setScrolled] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = () => {
```

Comment:
gg, now in later commits we'll use framer-motion useScroll() to get it and show progress in the acm logo container